packages/@aws-cdk/cloudformation-include/test/test-templates/functions.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
I think having test templates: `functions`, `conditions` and `functions-and-conditions` is a little much. Can we just have `functions-and-conditions`?